.phpcs.xml.dist

```diff
@@ -49,7 +49,7 @@
 			<!-- Provide the plugin specific prefix for use with namespace and hook names. -->
 			<property name="prefixes" type="array">
 				<element value="JoostBlog\WP\Comment"/>
```

Comment:
Should we change this namespace too? :)

Reply:
Yeah, we definitely should... I just didn't know what to make it. :laughing:  `EmiliaProjects`?

Reply:
Yeah, or just `Comment_Hacks` and not go so deep? Is there a reason to have the company name here?

Reply:
Done. Renamed `JoostBlog` to `EmiliaProjects` and for constants renamed `JOOST_*` to `EMILIA_*` (https://github.com/Emilia-Capital/comment-hacks/pull/117/commits/97437816608d023d0a1d00653497246404657e98)